sd_jwt/operations.py

```diff
@@ -0,0 +1,274 @@
+import datetime
+import logging
+import random
+from base64 import urlsafe_b64decode, urlsafe_b64encode
+from hashlib import sha256
+from json import dumps, loads
+from secrets import compare_digest
+from typing import Union
+
+from jwcrypto.jws import JWS
+from jwcrypto.jwk import JWK
+
+from sd_jwt.utils import pad_urlsafe_b64
+from sd_jwt.walk import by_structure as walk_by_structure
+from sd_jwt import DEFAULT_SIGNING_ALG, SD_CLAIMS_KEY, SD_DIGESTS_KEY, HASH_ALG_KEY
+
+DEFAULT_EXP_MINS = 15
+# TODO: adopt a dynamic module/package loader, defs could be as string -> "fn": "hashlib.sha256"
+HASH_ALG = {"name": "sha-256", "fn": sha256}
+SD_JWT_HEADER = "sd-jwt"
```

Comment:
```suggestion
SD_JWT_HEADER = "sd+jwt"
```

Reply:
At least it should not be in this PR. Ideally, one PR per issue.

Reply:
Done here **https://github.com/oauthstuff/draft-selective-disclosure-jwt/pull/61/commits/1bc26f545cbb5dd68496f2246c21231f4019e6d4**